fecgov/openFEC
# drop unused sched c d e f tables and related process

## Summary (required)
Clean up work to remove database objects that were replaced in issue #3089 to reduce complexity and redundancy of tables used.  External process and sql files used to refresh these objects are removed. 

- Resolves # [3100]
drop ofec_sched_e table and related process #3100

_Include a summary of proposed changes._
Clean up work to remove database objects that were replaced in issue #3089 to reduce complexity and redundancy of tables used.  External process and sql files used to refresh these objects are removed. 

## How to test the changes locally
Download this branch to local machine.  
Run create_sample_db
database objects listed in the migration file do not exist in the cfdm_test database
Sample data should load successfully without tables or columns not exist error.

## Impacted areas of the application
List general components of the application that this PR will affect:
None
-  



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
